### PR TITLE
Add finding projects resources to resource page

### PR DESCRIPTION
**Pull request by lucywyman**

### CODE REVIEW

File: source/resources.rst [hunk 1]
```diff
@@ -29,6 +29,19 @@ Systems Administration Tutorials
...
 
+Contributing
+~~~~~~~~~~~~
+
+* `Openhatch <http://openhatch.org/search>`_
+* `24 pull requests <http://24pullrequests.com/>`_
+* `BugsAhoy <http://www.joshmatthews.net/bugsahoy/>`_
+* `Showcased github projects <https://github.com/showcases>`__
+* `Trending github projects <https://github.com/trending>`_
+* Choose a company, search "$COMPANY open source"
```
Comment (jordane) [inline source/resources.rst]: Why a company?

Comment (lucywyman) [inline source/resources.rst]: s/company/organization/ ?

**Comment by lucywyman** [comment]
@ElijahCaine merge this?  I can also take out everything but the additions to the resources page.

**Comment by ElijahCaine** [comment]
@lucywyman merge it :+1:

**Comment by lucywyman** [comment]
That felt good.

**Pull request merged**